gatsby/content/projects/clients/matrix-commander.mdx

```diff
@@ -0,0 +1,172 @@
+---
+layout: projectimage
+title: matrix-commander
+categories:
+ - client
+description: Simple CLI-based Matrix client for sending and receiving with support for end-to-end encryption and emoji verification
+author: 8go
+home: https://github.com/8go/matrix-commander
+language: Python 3 (matrix-nio SDK)
```

Comment:
Please split this line to use `language: Python` and a separate `sdk` field, same as https://github.com/matrix-org/matrix.org/blob/master/gatsby/content/projects/2019-02-22-weechat-matrix.mdx

Reply:
I performed the requested change in commit #3 in 870b7d37cf056baa7144d0ec7cf93305ca9a48ad

Since the reference link had a lot more details, I added other fields as well, following the example of weechat.

Let me know if I can do anything else!

Reply:
> Do you have a Matrix room for the project?
No, I don't.